vpr/src/route/rr_graph_writer.cpp

```diff
@@ -37,6 +39,15 @@ void write_rr_segments(fstream& fp, const std::vector<t_segment_inf>& segment_in
 /* This function is used to write the rr_graph into xml format into a a file with name: file_name */
 void write_rr_graph(const char* file_name, const std::vector<t_segment_inf>& segment_inf) {
     fstream fp;
+    FILE* fb;
```

Comment:
As far as I can tell this won't work for binary.  It will write out as ascii text which is not what we want.  There's a possibility of changing over to fstream write() but I do not this this is much different from the current fwrite() and doesn't seem to be worth it for a temporary solution.

Reply:
Yeah, I believe you are right, using fstream would overcomplicate the solution to be able to write in bianry. As you say I think it could be kept as is for now.